users/tests/test_users_views.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from django.urls import reverse
+from django.test import Client
+from pytest_django.asserts import assertTemplateUsed
+from django.contrib.auth.models import User
+
+
+class TestViews:
+    def setup(self):
+        self.client = Client()
+
+    @pytest.fixture
+    def user(self):
+        return User.objects.create_user('Test-user', 'Test-user@mail.com', 'Password')
+
+    def test_registar_POST(self):
+        response = self.client.post(reverse('register'))
+        assert response.status_code == 200
+        assertTemplateUsed(response, 'users/register.html')
+
```

Comment:
can you change the name 'test_registar_POST' to 'test_register_POST' ?
same for 'test_registar_GET' 👍

Reply:
> can you change the name 'test_registar_POST' to 'test_register_POST' ?
> same for 'test_registar_GET' 👍

done.